MapboxNavigation/NavigationViewController.swift

```diff
@@ -281,6 +294,7 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
         super.viewWillAppear(animated)
         
         UIApplication.shared.isIdleTimerDisabled = true
+        styles.forEach { $0.apply() }
```

Comment:
Doesn't seem to work. I think this is a Swift 3.2 feature, looks more readable though.